buildSrc/src/main/groovy/com/wiredforcode/spawn/KillProcessTask.groovy

```diff
@@ -0,0 +1,23 @@
+package com.wiredforcode.gradle.spawn
+
+import org.gradle.api.tasks.TaskAction
+
+class KillProcessTask extends DefaultSpawnTask {
```

Comment:
what is this for?

Reply:
https://github.com/opensearch-project/sql/pull/507#discussion_r833802454

Commented here.